build.gradle

```diff
@@ -74,6 +74,12 @@ allprojects {
         resolutionStrategy {
             failOnVersionConflict()
         }
+
+        resolutionStrategy.eachDependency { details ->
+            if (details.requested.group == 'com.palantir.docker.compose' && details.requested.name == 'docker-compose-rule') {
+                details.useTarget group: 'com.palantir.docker.compose', name: 'docker-compose-rule-junit4', version: dependencyRecommendations.getRecommendedVersion('com.palantir.docker.compose', 'docker-compose-rule-junit4')
```

Comment:
Hmm, seems like this is needed because docker-proxy-rule still references the old docker-compose-rule (without -core and -junit4)